lib/matplotlib/tests/test_usetex.py

```diff
@@ -168,7 +168,7 @@ def test_rotation():
     mpl.rcParams['text.usetex'] = True
 
     fig = plt.figure()
-    ax = fig.add_axes([0, 0, 1, 1])
+    ax = fig.add_axes((0, 0, 1, 1))
     ax.set(xlim=[-0.5, 5], xticks=[], ylim=[-0.5, 3], yticks=[], frame_on=False)
```

Comment:
`ax.set` probably won't type check, but for consistency, `xlim` and `ylim` here should also be tuples.